### DISCUSSION TITLE
Add a favicon

### DISCUSSION THREAD

**Discussion by jtenniswood** [louislam/dockge#126] [Feature Request]
It will make finding the service easier in bookmarks and tabs.

**Comment by Zaid-maker** [comment]
There is a favicon?

![image](https://github.com/louislam/dockge/assets/53424436/f72b2f02-8291-4906-8a59-afc242b1390e)

**Reply by jtenniswood** [reply]
Oh weird, it's not loading for me on safari

**Reply by jtenniswood** [reply]
<img width="182" alt="Screenshot 2023-11-21 at 18 32 47" src="https://github.com/louislam/dockge/assets/10881541/2f662839-94f9-43ea-b66c-7784e15b8346">
